src/components/Header/Header.js

```diff
@@ -229,7 +229,7 @@ const Header = ({ setShowSearch, location, themeProvider, show, toggleFullscreen
         },
       } = data;
       const logoLink = logo.link !== '' ? logo.link : '/';
-      const logoImg = require('images/logo.svg');
+      const logoImg = require('images/dotnetthailand-logo.svg');
```

Comment:
Just curiosity, does import statement work for import resource file?
If so, could you please use import statement?